.github/workflows/release.yml

```diff
@@ -268,3 +268,14 @@ jobs:
         # a manual PR to edit the current CHANGELOG.next.toml file and remove the released entries.
         git -c 'user.name=AWS SDK Rust Bot' -c 'user.email=aws-sdk-rust-primary@amazon.com' commit CHANGELOG.next.toml --message "Remove released entries from \`CHANGELOG.next.toml\`"
         git push origin main
+
+  open-backport-pull-request:
+    name: Open backport pull request to merge the release branch back to main
+    needs:
+    - release
+    - trim-changelog-next-on-main
+    # See https://github.com/actions/runner/issues/2205#issuecomment-1381988186 for details on the workaround
+    if: inputs.dry_run == false && always() && contains(needs.*.result, 'success')
```

Comment:
**correctness**:

`contains(needs.*.result, 'success')` would be true if either dependent job is successful right? Is that what we want here?

Reply:
Good point. You know what, I don't think this job needs to depend on `release` because the other `trim-changelog-next-on-main` already depends on `release`. So it can just depend on `trim-changelog-next-on-main`.

Reply:
Addressed in 57b3d48